src/scores/categorical/contingency_impl.py

```diff
@@ -716,7 +716,7 @@ def precision(self) -> xr.DataArray:
         Notes:
             - Range: 0 to 1.  Perfect score: 1.
             - "True positives" is the same as "hits"
```

Comment:
```suggestion
            - "True positives" is the same as "hits".
```
I know this didn't have a full stop before, but I thought I would take the opportunity to suggest an update (simply for reasons of consistency).